examples/step-28/step-28.cc

```diff
@@ -158,9 +158,9 @@ namespace Step28
   //
   // At present, material data is stored for 8 different types of
   // material. This, as well, may easily be extended in the future.
-  MaterialData::MaterialData (const unsigned int n_groups)
+  MaterialData::MaterialData (const unsigned int n_groups_)
     :
-    n_groups (n_groups),
+    n_groups (n_groups_),
```

Comment:
It's doing the exact same thing as your change but without changing the name of the input argument.